core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -126,9 +126,13 @@ var PreferencesPage = function() {
   };
 
   this.setUserInterests = async function(interests) {
-    await userInterestsInput.click();
+    await action.click('User Interest Input', userInterestsInput);
     for (var i = 0; i < interests.length; i++) {
-      await userInterestsInput.sendKeys(interests[i], protractor.Key.RETURN);
+      await action.sendKeys(
+        'User Interest Input', userInterestsInput, interests[i]);
+      await action.sendKeys(
+        'User Interest Input', userInterestsInput,
+        protractor.Key.RETURN);
       await saveNewChanges('User Interests');
     }
   };
```

Comment:
Can you wait for `subscriptions.first()` to be visible before getting its text in `this.expectDisplayedFirstSubscriptionToBe` below? Ditto for all other `.getText()` calls below

Reply:
OK